Blog CSS

ShCiv opened the pull request:
На проверку.

ShCiv committed:
Add files via upload

На проверку.

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,121 @@
+body {
+  background: linear-gradient(180deg, #374151 0%, #111827 100%);
+  width: 1440px;
+  height: 1835px;
+  margin: 0%;
```
Ну height лучше не выставлять, а вдруг контента будет больше?
а ширину лучше сделать через max-width, т.е. не фактический размер, а не более
и чтобы выставляло по центру экрана - сделать margin: 0 auto (где 0 - top & bottom), auto - left & right пополам

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,121 @@
+  margin: 0%;
+}
+
+ul a {
+  color: #ffffff;
+  font-family: Roboto;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 20px;
+  text-decoration: none;
+}
```
что касается ссылок - лучше вынести в стилизацию "a" отдельно, т.к. мы получили в лого ссылку без "стилизации"
а для ul - хорошо бы list-style: none указать, т.к. у нас есть "точки" стилизации и сразу же убрать базовый padding

![image](https://github.com/ShCiv/DZ/assets/101252030/5c5f5210-c13d-43d9-ba07-bccb873a24b3)
![image](https://github.com/ShCiv/DZ/assets/101252030/f65c5f77-376d-4ea4-b690-da7e5c96b2a4)

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,121 @@
+
+input {
+  font-family: Roboto;
+  font-size: 24px;
```
многовато, не находите?)
по макету 16px
Так же можно сразу указать border: none; (рамка) и outline:none (внутренняя рамка)

GitEsense reviewed (changes requested):
Оставил комментарии на исправления, так же отсутствуют сами изображения, и полностью отсутствуют стили для файла формы (2-Form) - её тоже бы стилизовать, а так всё в целом отлично

ShCiv committed:
Form

Сделала, но не отправила... исправляюсь

ShCiv committed:
Add files via upload

Сделаны исправления

ShCiv committed:
IMG

ShCiv committed:
IMG

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,127 @@
+}
+
+.blogFoto {
+  margin: 0 50%;
```
margin 50% сделает 50% с каждой стороны от размера экрана, если хотим в центр экрана, то лучше использовать auto (он поделить свободное поровну на обе стороны)

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,127 @@
+
+.blogFoto {
+  margin: 0 50%;
+  width: 50px;
+  height: 50px;
```
лучше использовать max-width / max-height , т.к. лучше не ограничивать жёстко, если это не требуется, а так мы сможем "динамику" дать, но не превышать

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,127 @@
+  margin: 0 50%;
+  width: 50px;
+  height: 50px;
+  top: 316px;
+  left: 640px;
```
Вообще не будут работать в данном случае

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,127 @@
+.colorFoto {
+  width: 896px;
+  height: 594px;
+  top: 397px;
+  left: 272px;
+  border-radius: 5px;
+  margin: 27px 272px 69px;
```
аналогично, что и выше

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,127 @@
+  text-align: center;
+}
+.colorFoto {
+  width: 896px;
+  height: 594px;
```
а вот тут жёсткий размер будет как раз

GitEsense commented on 1-Blog/style.css:
```diff
@@ -0,0 +1,127 @@
+  width: 708px;
+  text-align: justify;
+  color: #ffffff;
+  font-family: Roboto;
+  font-size: 16px;
+  font-weight: 400;
```
можно вынести в body и менять лишь размер шрифта по необходимости в нужном блоке

GitEsense commented on 2-Form/style.css:
```diff
@@ -0,0 +1,129 @@
+  background: linear-gradient(244deg, #fdc5c5, #ffa5a5);
+}
+
+main {
+  background-color: #fff;
+  width: 582px;
+  height: 573px;
+  border-radius: 10px;
+  margin: 225px 429px;
+  box-shadow: 0px 4px 4px 0px #000000 25%;
+}
```
т.к. у нас main - это средняя часть формы - правильнее было бы использовать эти стили на form, и избавиться от страшного марджина, так же исправить box-shadow, он с ошибкой

GitEsense commented on 2-Form/style.css:
```diff
@@ -0,0 +1,129 @@
+body {
+  width: 1440px;
+  height: 1024px;
+  margin: 0%;
```
```css
  max-width: 1440px;
  margin: 0;
```
незачем высоту ограничивать и марджин если 0, то 0, без процентов

GitEsense reviewed (changes requested):
Необходимо положить картинки, перепроверить ссылки, добавить в "head" формы всё-таки стили (файл не подключен)
padding формы можно установить на всю форму, тем самым все элементы будут иметь общий внешний (внутренний для формы) отступ

ShCiv committed:
Исправленная версия

Добавила картинку стрелочки, сдалал исправления, не могу разобраться почему чекбокс отображается не так как задано.

ShCiv committed:
Исправления

Рассмотрела все замечания, кажется всё подправила, начала разбираться с форматом отображения блочных и строчных данных.. кажется.

GitEsense commented on 2-Form/style.css:
```diff
@@ -1,17 +1,16 @@
-  margin: 0%;
+  max-width: 1440px;
+  margin: 0;
   background: linear-gradient(244deg, #fdc5c5, #ffa5a5);
```
```css
background: linear-gradient(
            225deg,
            #fdc5c5 25.45%,
            rgba(242, 152, 222, 0) 100%
        ),
        linear-gradient(225deg, #ffa5a5 33.09%, rgba(255, 255, 255, 0) 82.34%),
        #f39494;
```
для корректного

GitEsense reviewed (approved):
Всё отлично)
бг цвет для формы я Вам положил в комментарий, его можно будет использовать (это 100% соответствующий макету)
